src/cli/global/install.rs

```diff
@@ -52,6 +52,10 @@ pub struct Args {
 
     #[clap(flatten)]
     config: ConfigCli,
+
+    /// Specifies that the packages should be reinstalled even if they are already installed.
+    #[arg(action, short, long)]
```

Comment:
```suggestion
    #[arg(action, long)]
```
Let's not make this work with `-f` yet, this might be confusing with the `features`. It's also not a daily command so i don't mind longer names.